server/service/individualServices/SoftwareUpgrade.js

```diff
@@ -316,8 +325,8 @@ async function PromptForBequeathingDataCausesRObeingRequestedToStopNotifications
             let forwardingKindNameOfTheBequeathOperation = "PromptForBequeathingDataCausesRObeingRequestedToStopNotificationsToOldRelease";
 
             let listOfOperationToBeUnsubscribed = [];
-            let approvalOperationName = await operationClientInterface.getOperationNameAsync("aa-0-0-1-op-c-3020");
-            let deregistrationOperationName = await operationClientInterface.getOperationNameAsync("aa-0-0-1-op-c-3021");
+            let approvalOperationName = await operationClientInterface.getOperationNameAsync("aa-2-0-1-op-c-im-ro-2-0-1-000");
```

Comment:
Please fetch the operationClientUuids from the forwardings instead of hardcoding